src/ui/public/filter_manager/lib/__tests__/range.js

```diff
@@ -29,18 +30,14 @@ describe('Filter Manager', function () {
     });
 
     it('should return a script filter when passed a scripted field', function () {
-      expected.meta.field = 'script number';
-      _.set(expected, 'script.script', {
-        lang: 'expression',
-        inline: '(' + indexPattern.fields.byName['script number'].script + ')>=gte && (' +
-        indexPattern.fields.byName['script number'].script + ')<=lte',
-        params: {
-          value: '>=1 <=3',
-          gte: 1,
-          lte: 3
-        }
-      });
-      expect(fn(indexPattern.fields.byName['script number'], {gte: 1, lte: 3}, indexPattern)).to.eql(expected);
+      const filter = fn(indexPattern.fields.byName['script number'], {gte: 1, lte: 3}, indexPattern);
+      const expectedInline = '(' + indexPattern.fields.byName['script number'].script + ')>=gte && (' +
+        indexPattern.fields.byName['script number'].script + ')<=lte';
+
+      expect(filter.script.script.lang).to.eql('expression');
+      expect(filter.script.script.inline).to.eql(expectedInline);
+      expect(filter.script.script.params.value).to.eql('>=1 <=3');
+      expect(filter.meta.field).to.eql('script number');
```

Comment:
I'm reverting this change, indeed it's not needed.  I arrived here because initially I updated the field to match non-scripted, so it was "[start date] to [end date]" rather than ">=[start date] <=[end date]".  When the test failed the params were out of order, a side affect I thought of my updated code and then I thought, how silly that the order of params matters, so adjusted the check to be like this.  On second thought though, it must have failed because of the string comparison, not because of the order of params (I'm guessing the comparator is smart enough to handle that).  Since I reverted the new format I'm reverting this whole file.